index.d.ts

```diff
@@ -202,7 +202,7 @@ interface LoginOptions {
   authorizationParams?: AuthorizationParameters;
 
   /**
-   *  URL to return to after login, overrides the Default is {@link Request.originalUrl}
+   *  URL to return to after login, overrides the Default is {@link express!Request.originalUrl Request.originalUrl}
```

Comment:
This now links externally to the express docs based on `externalSymbolLinkMappings` in typedoc.js